### PR TITLE
Add Wine 4.0 on Debian

### PR DESCRIPTION
**Pull request by wgos**
The image supports:

- steamcmd
- Beta access
- Auto update Steam application
- Changing wineprefix
- Installing different dlls/apps/settings via winetricks
- Changing Xvfb parameters such as Display height/wight, color depth
- Disabling/enabling Xvfb

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+if [[ $XVFB == 1 ]]; then
+        Xvfb :0 -screen 0 ${DISPLAY_WIDTH}x${DISPLAY_HEIGHT}x${DISPLAY_DEPTH} &
+fi
+
+# Install necessary to run packages
+echo "First launch will throw some errors. Ignore them"
+
+if [ ! -d $WINEPREFIX ]; then
+        mkdir $WINEPREFIX
```
Comment (parkervcp) [inline entrypoint.sh]: `mkdir -p` will run and not throw errors a lot of the time

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+fi
+
+# List and install other packages
+for trick in $WINETRICKS_RUN; do
+        echo "Installing $trick"
+        winetricks -q $trick
+done
+
+# Replace Startup Variables
+MODIFIED_STARTUP=`eval echo $(echo ${STARTUP} | sed -e 's/{{/${/g' -e 's/}}/}/g')`
```
Comment (parkervcp) [inline entrypoint.sh]: Can you try to use the same startup as this one here - https://github.com/parkervcp/images/blob/bot/discordjs/entrypoint.sh#L8

Comment (wgos) [reply entrypoint.sh]: Sure

**Comment by wgos** [comment]
Applied your advices

File: README.md [hunk 3]
```diff
@@ -1,2 +1,3 @@
-# Containers
-Generic docker containers designed to work with Pterodactyl Panel and its daemon.
+## Wreckfest
```
Comment (parkervcp) [inline README.md]: This is not specific to wreckfest anymore so it should be the same generic image README

**Comment by parkervcp** [comment]
@WGOS Please check my review.

**Comment by parkervcp** [comment]
@WGOS Please respond to my review I requested on Nov 3rd.

**Review by wgos — COMMENTED** [review]
Seems like checked

**Review by parkervcp — APPROVED** [review]

**Comment by parkervcp** [comment]
All good now. you can move forward on the wreckfest egg.

File: README.md (hunk 3 above)
Comment (wgos) [reply README.md]: Already done

**Comment by wgos** [comment]
> All good now. you can move forward on the wreckfest egg.

Where could I find docker image of this repo?

**Comment by parkervcp** [comment]
All my images are built in quay at `quay.io/parkervcp/pterodactyl-images` The tag for this image is `debian_wine-4`

Complete image would be `quay.io/parkervcp/pterodactyl-images:debian_wine-4`

**Comment by wgos** [comment]
Thank you

**Pull request merged**